tests/src/JIT/Performance/RunBenchmarks/RunBenchmarks.cmd

```diff
@@ -0,0 +1,38 @@
+@setlocal
+@echo off 
+
+set ARCH=%1
+set BUILD=%2
+
+if /I "%ARCH%" == "" set ARCH=x64
+if /I "%BUILD%" == "" set BUILD=Release
+
+rem *** set this appropriately for enlistment you are running benchmarks in
+if /I NOT "%CORECLR_ROOT%" == "" goto over
```

Comment:
If CORECLR_ROOT isnt' set, we should be able to set it off of the sensible default for arch/build, then test whether it exists.